src/main/java/edu/ucsb/mapache/entities/UserSearch.java

```diff
@@ -22,6 +22,8 @@
   private String searchTerm;
```

Comment:
In order for your change to the repository to work, i.e.
```
public List<UserSearch> findByEmail(String email);
```

You also have to change the userID field to be `email`

Reply:
Hi Professor Conard
 I am not sure is it a problem we just meet because I think our heroku and local searches are working well.
 We tried to add a new variable email in the UserSearch.java on line 26. It was also not working at first after deploying. However, we found the issue that the old database is conflicting with the new database because it does not have the entity email. After we delete the old database and rerun the code, it works. So basically we didn't replace the userID by email and just add a new entity. Would this work? If not I will follow your instruction to replace it.
![image](https://user-images.githubusercontent.com/49249651/120531959-c562b280-c411-11eb-8385-fbe651a9b49d.png)

Reply:
As shown in the following pictures, I think our search is working on heroku? 
![image](https://user-images.githubusercontent.com/49249651/120532432-3efaa080-c412-11eb-9aaf-fe3c247d24bd.png)
![image](https://user-images.githubusercontent.com/49249651/120532524-52a60700-c412-11eb-8742-1b944130793a.png)
![image](https://user-images.githubusercontent.com/49249651/120532558-5afe4200-c412-11eb-9281-cb230773213e.png)